src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueStateMachineTests.cs

```diff
@@ -598,500 +598,483 @@ static Task<int> F()
-
-            // Verify that no new TypeDefs, FieldDefs or MethodDefs were added,
-            // 3 methods were updated: 
-            // - the kick-off method (might be changed if the method previously wasn't an iterator)
-            // - Finally method
-            // - MoveNext method
-            CheckEncLogDefinitions(md1.Reader,
-                Row(3, TableIndex.StandAloneSig, EditAndContinueOperation.Default),
-                Row(1, TableIndex.MethodDef, EditAndContinueOperation.Default),
-                Row(4, TableIndex.MethodDef, EditAndContinueOperation.Default),
-                Row(5, TableIndex.MethodDef, EditAndContinueOperation.Default),
-                Row(1, TableIndex.CustomAttribute, EditAndContinueOperation.Default),
-                Row(7, TableIndex.CustomAttribute, EditAndContinueOperation.Default));
-
-            diff1.VerifyIL("C.<F>d__0.System.Collections.IEnumerator.MoveNext", @"
-{
-  // Code size       57 (0x39)
-  .maxstack  2
-  .locals init (int V_0)
-  IL_0000:  ldarg.0
-  IL_0001:  ldfld      ""int C.<F>d__0.<>1__state""
-  IL_0006:  stloc.0
-  IL_0007:  ldloc.0
-  IL_0008:  brfalse.s  IL_0012
-  IL_000a:  br.s       IL_000c
-  IL_000c:  ldloc.0
-  IL_000d:  ldc.i4.1
-  IL_000e:  beq.s      IL_0014
-  IL_0010:  br.s       IL_0016
-  IL_0012:  br.s       IL_0018
-  IL_0014:  br.s       IL_0030
-  IL_0016:  ldc.i4.0
-  IL_0017:  ret
-  IL_0018:  ldarg.0
-  IL_0019:  ldc.i4.m1
-  IL_001a:  stfld      ""int C.<F>d__0.<>1__state""
-  IL_001f:  nop
-  IL_0020:  ldarg.0
-  IL_0021:  ldc.i4.2
-  IL_0022:  stfld      ""int C.<F>d__0.<>2__current""
-  IL_0027:  ldarg.0
-  IL_0028:  ldc.i4.1
-  IL_0029:  stfld      ""int C.<F>d__0.<>1__state""
-  IL_002e:  ldc.i4.1
-  IL_002f:  ret
-  IL_0030:  ldarg.0
-  IL_0031:  ldc.i4.m1
-  IL_0032:  stfld      ""int C.<F>d__0.<>1__state""
-  IL_0037:  ldc.i4.0
-  IL_0038:  ret
-}");
-            v0.VerifyIL("C.<F>d__0.System.Collections.IEnumerator.MoveNext", @"
-{
-  // Code size       57 (0x39)
-  .maxstack  2
-  .locals init (int V_0)
-  IL_0000:  ldarg.0
-  IL_0001:  ldfld      ""int C.<F>d__0.<>1__state""
-  IL_0006:  stloc.0
-  IL_0007:  ldloc.0
-  IL_0008:  brfalse.s  IL_0012
-  IL_000a:  br.s       IL_000c
-  IL_000c:  ldloc.0
-  IL_000d:  ldc.i4.1
-  IL_000e:  beq.s      IL_0014
-  IL_0010:  br.s       IL_0016
-  IL_0012:  br.s       IL_0018
-  IL_0014:  br.s       IL_0030
-  IL_0016:  ldc.i4.0
-  IL_0017:  ret
-  IL_0018:  ldarg.0
-  IL_0019:  ldc.i4.m1
-  IL_001a:  stfld      ""int C.<F>d__0.<>1__state""
-  IL_001f:  nop
-  IL_0020:  ldarg.0
-  IL_0021:  ldc.i4.1
-  IL_0022:  stfld      ""int C.<F>d__0.<>2__current""
-  IL_0027:  ldarg.0
-  IL_0028:  ldc.i4.1
-  IL_0029:  stfld      ""int C.<F>d__0.<>1__state""
-  IL_002e:  ldc.i4.1
-  IL_002f:  ret
-  IL_0030:  ldarg.0
-  IL_0031:  ldc.i4.m1
-  IL_0032:  stfld      ""int C.<F>d__0.<>1__state""
-  IL_0037:  ldc.i4.0
-  IL_0038:  ret
-}");
+            using var _ = new EditAndContinueTest()
```

Comment:
>using var _ = new EditAndContinueTest() [](http://example.com/codeflow?start=12&length=39)

Are the changes to this test (and following tests) simply to write the tests using the current format, or are there behavior changes as well?

Reply:
The new test helpers construct proper syntax map. The previous one did not construct it and it worked only accidentally.